scripts/script.js

```diff
@@ -28,6 +28,10 @@ function serialize_object(x) {
         return "";
 }
 
+//alerts success of sharing with Open Academics
+function share_success() {
+    alert("You have successfully shared your results with search engines.");
+}
```

Comment:
Nah. Do this a different way. Don't want it in the main script, and alerts suck.